src/iceberg_explorer/api/routes/catalog.py

```
@@ -0,0 +1,458 @@
+
+    Args:
+        table_path: Table path like 'accounting.transactions' or 'accounting\\x1ftax.records'
+
+    Returns:
+        Tuple of (namespace_parts, table_name)
+
+    Raises:
+        ValueError: If the path format is invalid.
+    """
+    if "." not in table_path:
+        raise ValueError(f"Invalid table path format: {table_path}. Expected 'namespace.table'")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)